### PR TITLE
change  shopping cart to minicart

### PR DESCRIPTION
**Pull request by sunilit42**
Fixed issue :- #24409

### CODE REVIEW

**Review by XxXgeoXxX — CHANGES_REQUESTED** [review]
Magento has a lot of additional entities that relevant to this label.
We could not change this label only in one place and skip others.
Please, recheck and update translations, MFTF tests, comment, etc to prevent misunderstanding.

**Comment by sunilit42** [comment]
@XxXgeoXxX 

thanks for the review .

I did into translate csv file and I checked into test folder nothing related to this.

File: app/code/Magento/Checkout/i18n/en_US.csv [hunk 1]
```diff
@@ -156,8 +156,8 @@ Shipping,Shipping
 "Number of Items to Display Pager","Number of Items to Display Pager"
 "My Cart Link","My Cart Link"
 "Display Cart Summary","Display Cart Summary"
-"Shopping Cart Sidebar","Shopping Cart Sidebar"
-"Display Shopping Cart Sidebar","Display Shopping Cart Sidebar"
+"Mini Cart","Mini Cart"
```
Comment (XxXgeoXxX) [inline app/code/Magento/Checkout/i18n/en_US.csv]: This translate has already existed. It can be removed.

**Review by XxXgeoXxX — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @XxXgeoXxX, thank you for the review.
[ENGCOM-5752](https://jira.corp.magento.com/browse/ENGCOM-5752) has been created to process this Pull Request
:eight_spoked_asterisk: @XxXgeoXxX, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
Hi @sunilit42 There are left few occurrences of "Shopping Cart Sidebar" in the comments:
image



Could you take a look?

**Comment by sunilit42** [comment]
> Hi @sunilit42 There are left few occurrences of "Shopping Cart Sidebar" in the comments:
> image
> 
> Could you take a look?

I did please check it, I did not do for one file because it is related to admin shopping  cart

**Review by XxXgeoXxX — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @XxXgeoXxX, thank you for the review.
[ENGCOM-5752](https://jira.corp.magento.com/browse/ENGCOM-5752) has been created to process this Pull Request
:eight_spoked_asterisk: @XxXgeoXxX, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Pull request merged**